lian/src/lian_nav2/src/mission.cpp

```diff
@@ -3,22 +3,23 @@
 #include "std_msgs/msg/string.hpp"
 #include "nav_msgs/msg/Path.hpp"
 #include "geometry_msgs/PoseStamped.hpp"
+#include "nav_msgs/msg/OccupancyGrid.hpp"
```

Comment:
Writing here because github doesn't let to write where no code was changed. We need to get rid of all the logger related logic in this file, and just use ROS's logger. You can pass it to `mission`'s constructor. We want to have a single logger system-wide.

Reply:
got it